org_fedora_hello_world/dbus_service/kickstart.py

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+from pykickstart.options import KSOptionParser
+
+from pyanaconda.core.kickstart import VERSION, KickstartSpecification
+from pyanaconda.core.kickstart.addon import AddonData
+
+import logging
+
+log = logging.getLogger(__name__)
+
+# The kickstart.py file of a service/module defines the parts needed for handling Kickstart data in
+# the service.
```

Comment:
I am not sure how much we want to use these types of comments. Maybe I would change them to module docstrings, so they are eventually exportable.

Reply:
Changed to docstring.